frontend/src/pages/notebookController/screens/server/NotebookServer.tsx

```diff
@@ -40,20 +49,12 @@ export const NotebookServer: React.FC = React.memo(() => {
         {notebook && (
           <div className="odh-notebook-controller__page">
             <ActionList>
-              <ActionListItem
-                onClick={() => {
-                  deleteNotebook(projectName, generateNotebookNameFromUsername(username))
-                    .then(() => {
-                      history.push(`/notebookController/spawner`);
-                    })
-                    .catch((e) => console.error(e));
-                }}
-              >
+              <ActionListItem onClick={() => setNotebooksToStop([notebook])}>
                 <Button variant="primary">Stop notebook server</Button>
               </ActionListItem>
               <ActionListItem
                 onClick={() => {
-                  if (notebook?.metadata.annotations?.['opendatahub.io/link']) {
+                  if (notebook.metadata.annotations?.['opendatahub.io/link']) {
```

Comment:
Isn't notebook still optional?

Reply:
https://github.com/opendatahub-io/odh-dashboard/pull/398/files#diff-6044aa8ded174b72428e0b2c0af57d57944b3f5eb055a04802c14fb9588b867aR49 the check here guarantee the notebook always exists.

Reply:
DOH, that's what I get for not expanding the code or looking at it in my IDE haha. 👍